composer.json

```diff
@@ -100,7 +101,8 @@
         "symfony/browser-kit": "^4.4 || ^5.2",
         "symfony/http-client": "^4.4 || ^5.2",
         "symfony/phpunit-bridge": "^5.2",
-        "symfony/security-csrf": "^4.4 || ^5.2"
+        "symfony/security-csrf": "^4.4 || ^5.2",
+        "vimeo/psalm": "^4.7.2"
```

Comment:
We should also add the plugin for phpunit.

Reply:
Solved